### DISCUSSION TITLE
Installing JDBC Driver... issues with chatGPT

### DISCUSSION THREAD

**Discussion by CDAliens88** [dbeaver/dbeaver#19415] [Q&A]
I have recently downloaded the Enterprise version of DBeaver as I wish to use the new ChatGPT functionality... however in order to do so, it's my understanding that I need to install a JDBC driver and install.

I have verified that I have Java installed on my computer (followed these directions)  https://www.howtogeek.com/812583/how-to-open-jar-files-windows/#:~:text=To%20run%20a%20JAR%20file,in%20a%20program%20like%20WinRAR.

I am attempting to follow the directions here - https://docs.informatica.com/integration-cloud/data-integration-connectors/current-version/mysql-connector/introduction-to-mysql-connector/administration-of-mysql-connector/installing-mysql-jdbc-driver.html

I am using MYSQL so downloaded the mysql-connector-j-8.0.32 "executable Jar file" and have extracted everything. When I "open with" and select "open with Java platform SE"... nothing happens. I'm not sure of next steps, as I'm assuming I need to run that connector file before I can move forward. 

Any help is appreciated thank you.

**Comment by E1izabeth** [comment]
Have you tried just creating a new connection to your database with DBeaver?

**Comment by CDAliens88** [comment]
Hello, thank you for the reply and the support.

Yes, I just created a new connection to the database. The connection is
live but the chatGPT is still giving the same error. I am using DBeaver
Enterprise 23.0.0. A screenshot of the AI (GPT Chat) settings is attached.
I'm using the proper key from my openai account.

![image](https://user-images.githubusercontent.com/128101584/226121073-ac350897-9508-4f9d-8e5f-6b46e3c8156a.png)
![image](https://user-images.githubusercontent.com/128101584/226121138-7c204f45-dc96-446e-9175-eeccf5f1d2dc.png)

**Reply by E1izabeth** [reply]
You get timeout error, because chatGPT servers are overloaded. It's not a DBeaver issue, just so many people are trying to use chatGPT at the same time as you.
